stable/prometheus-cloudwatch-exporter/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: "0.5.0"
+description: A Helm chart for prometheus cloudwatch-exporter
+name: prometheus-cloudwatch-exporter
+version: 0.4.1
+home: https://github.com/prometheus/cloudwatch_exporter
+sources:
+- https://github.com/prometheus/cloudwatch_exporter
+keywords:
+- aws
+- cloudwatch
+- prometheus
+- exporter
+maintainers:
+- email: gianrubio@gmail.com
```

Comment:
who is that?

Reply:
its the name of original maintainer. I refactor already existed chart, but now i change maintainer because i create PR to helm/charts repo with the same changes